troubleshooting/faq.md

```diff
@@ -162,6 +162,28 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## Why to use npm Enterprise?
+
+1. npm Enterprise is single tenant vs. multi tenant -- which is important for some compliance heavy companies; npm Enterprise also allows you to have any number of scopes, and provides its own website, which can be useful for large organizations.
+
+2. npm Enterprise allows you to run npm’s infrastructure behind your company’s firewall.
+
+4. Security is a serious concern, and with npm Enterprise we have mitigated the risks by partnering with *Node Security Platform* to provide the security features many enterprise customers have been looking for.
+
+   You can read our [blog](http://blog.npmjs.org/post/146943134240/npm-add-ons) about it, and another article straight from the [Node Security Platform](https://medium.com/node-security/announcing-npm-enterprise-security-add-on-6dde303efb9f).
+
+5. npm Enterprise can significantly improve the efficiency of the private code that you write at your organization making it easier to share documentation and code, distribute code to your build servers, and to breakup your monolithic code-bases into easier-to-reason-about modules.
```

Comment:
* omit "npm Enterprise can" from the beginning
* change "the efficiency of the private code that you write at your organization" to "the efficiency of your development process"
* change "distribute code to your build servers" to "streamline your build process"
* change "easier-to-reason-about modules" to "individual packages that are easier to maintain."